Misc/ACKS

```diff
@@ -1850,6 +1850,7 @@ Nikita Vetoshkin
 Al Vezza
 Petr Viktorin
 Jacques A. Vidrine
+Jeremiah Vivian (Pascual)
```

Comment:
Please try to maintain alphabetical order. Looks like this belongs below "Frank Visser" below.

Reply:
Fixed.